src/lib/i18n/locales/ca-ES/translation.json

```diff
@@ -408,8 +408,8 @@
 	"Model not selected": "Model no seleccionat",
 	"Model Params": "Paràmetres del model",
 	"Model updated successfully": "Model actualitzat correctament",
-	"Model Whitelisting": "Llista blanca de models",
-	"Model(s) Whitelisted": "Model(s) a la llista blanca",
```

Comment:
For the other languages, I don't know what the relevant term would be so not changing those in this PR